server/helpers/toyProblemsChecker.js

```diff
@@ -12,25 +11,59 @@ const getStudentsList = cohortName => {
   return studentsList && studentsList.length ? studentsList[0].students : [];
 };
 
-const checkIfPrTitleMatches = prTitle => {
-  return allToyProblems.some(substring => prTitle.toLowerCase().includes(substring));
+const getReleasedToyProblems = async cohort => {
+  const response = await githubQuery(
+    `https://api.github.com/repos/hackreactor/${cohort}-toy-problems/commits?per_page=100`
+  );
+
+  let releasedProblems = [];
```

Comment:
consider a set instead of an array to remove the need to filter duplicates.

Reply:
@magee I checked the function. `releasedProblems` is an array of objects with name and release date. The release date might be different for duplicate releases. For this reason, I could not use `set`